buzsaki_lab_to_nwb/original-code/convert_yuta.py

```diff
@@ -55,7 +51,7 @@ def get_UnitFeatureCell_features(fpath_base, session_id, session_path, max_shank
     """
 
     cols_to_get = ('fineCellType', 'region', 'unitID', 'unitIDshank', 'shank')
-    matin = loadmat(os.path.join(fpath_base, 'DG_all_6__UnitFeatureSummary_add.mat'),
+    matin = loadmat(os.path.join(fpath_base,'_extra/DG_all_6/DG_all_6__UnitFeatureSummary_add.mat'),
```

Comment:
This change adjusts the file location to match what is on the Globus endpoint; granted, the specific `DG_all_6` folder still had to be requested and added manually.